benchmarks/source-flotiq/gatsby-config.js

```diff
@@ -0,0 +1,27 @@
+require('dotenv').config();
+
+module.exports = {
+  siteMetadata: {
+    siteTitle: `Gatsby Flotiq Benchmark`,
+  },
+  plugins: [
+    `gatsby-plugin-benchmark-reporting`,
+    `gatsby-plugin-sharp`,
+    {
+      resolve: 'gatsby-source-flotiq',
+      options: {
+        baseUrl: "https://api.flotiq.com",
+        authToken: process.env.BENCHMARK_FLOTIQ_API_TOKEN,
+        forceReload: false,
+      },
+    },
+    `gatsby-transformer-sharp`,
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        name: `pages`,
+        path: `${__dirname}/src/pages/`,
+      },
+    },
```

Comment:
```suggestion
```

I'm not sure why this is here, gatsby handles pages in `src/pages` automatically. What was the reason this was put here?

Reply:
Copied it from one of the examples. Removed along with other updates in the next commit.